pages/index.js

```diff
@@ -1,22 +1,46 @@
-import Button from "@/components/Button";
-import styled from "styled-components";
+import styled, { keyframes } from "styled-components";
+import Link from "next/link";
 
 export default function HomePage() {
   return (
-    <>
-      <IntroContainer>
-        <IntroParagraph>
-          How much money can you save with a solar panel?
-        </IntroParagraph>
-        <IntroSubtext>Click below to find out!</IntroSubtext>
-      </IntroContainer>
-      <ButtonContainer>
-        <Button href="/radiation" text="Start" />
-      </ButtonContainer>
-    </>
+    <IntroContainer>
+      <IntroParagraph>
+        How much money can you save with a solar panel?
+      </IntroParagraph>
+      <IntroSubtext>Click image below to find out!</IntroSubtext>
+      <StyledLink href="/radiation">
+        <StyledImage
+          src="https://images.unsplash.com/photo-1509391366360-2e959784a276?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1172&q=80"
+          alt="Solar Panel"
+        />
```

Comment:
Add width and height here instead:
```suggestion
        <StyledImage
          src="https://images.unsplash.com/photo-1509391366360-2e959784a276?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1172&q=80"
          alt="Solar Panel"
          width="200"
          height="200"
        />
```